modules/storages/app/common/storages/peripherals/storage_interaction/one_drive/files_info_query.rb

```diff
@@ -44,19 +44,11 @@ def initialize(storage)
           end
 
           def call(auth_strategy:, file_ids:)
-            if file_ids.nil?
-              return ServiceResult.failure(
-                result: :error,
-                errors: ::Storages::StorageError.new(code: :error, log_message: "File IDs can not be nil")
-              )
-            end
-
-            result = file_ids.map do |file_id|
+            result = Array(file_ids).map do |file_id|
```

Comment:
🔴 this now has a changed behavior: if `file_ids` is nil, it now returns success with an empty result. Why the change?

Reply:
Oh, crap. I forgot to revert that. xD